pkg/optimizely/client.go

```diff
@@ -175,6 +175,22 @@ func (c *OptlyClient) ActivateFeature(key string, uc entities.UserContext, disab
 		return &Decision{}, err
 	}
 
+	var experimentKey, variationKey string
+	projectConfig, _ := c.OptimizelyClient.ConfigManager.GetConfig()
+
+	feature, _ := projectConfig.GetFeatureByKey(key)
```

Comment:
As per comment from PR # 224, please use OptimizelyConfig.

Reply:
Since we already need `projectConfig` to create `decisionContext`, we are just reusing it here for `feature`.